lib/Styler/widgets/FilterBuilder.js

```diff
@@ -1107,39 +1117,75 @@ Styler.FilterBuilder = Ext.extend(Ext.Panel, {
                                     alert("Cannot access XML filters");
                                 },
                                 success: function(response) {
-                                    var filterString = response.responseText,
-                                        fx = new OpenLayers.Format.XML(),
-                                        ff = new OpenLayers.Format.Filter.v2_0_0(),
-                                        loadedFilters = ff.readChildNodes(fx.read(filterString)),
-                                        clonedFilters = this.cloneFilter(loadedFilters.filter);
-
-                                    var a = {
+                                    var filterString, fx, ff, loadedFilters, clonedFilters, xmlFilters;
```

Comment:
The previous var declaration was OK (at least to me)... what did codacy say about it ?